signer/src/code_cid.rs

```diff
@@ -0,0 +1,47 @@
+use lazy_static::lazy_static;
+use serde_json::{json, Value};
+
+lazy_static! {
+    pub static ref ACTOR_CODE_CIDS: Value = json!(
```

Comment:
later on you can change this to grab the json from [here](https://github.com/Zondax/filecoin-actors-cids/releases/download/v1.1600.1/builtin-actors-cids_V8.json)